### PR TITLE
container_engine_t: another round of small improvements

### PR DESCRIPTION
**Pull request by haircommander**

### CODE REVIEW

**Comment by haircommander** [comment]
added one more piece PTAL @rhatdan

File: container.te [hunk 1]
```diff
@@ -1450,11 +1450,13 @@ allow container_engine_t sysctl_t:{dir file} mounton;
 allow container_engine_t fusefs_t:dir { relabelfrom relabelto };
 allow container_engine_t fusefs_t:file relabelto;
 allow container_engine_t kernel_t:system module_request;
-allow container_engine_t null_device_t:chr_file mounton;
+allow container_engine_t null_device_t:chr_file { mounton setattr };
```
Comment (rhatdan) [inline container.te:1453]: allow container_engine_t null_device_t:chr_file { mounton setattr_chr_file_perms };

File: container.te [hunk 2]
```diff
@@ -1450,11 +1450,13 @@ allow container_engine_t sysctl_t:{dir file} mounton;
...
 allow container_engine_t random_device_t:chr_file mounton;
 allow container_engine_t self:netlink_tcpdiag_socket nlmsg_read;
 allow container_engine_t urandom_device_t:chr_file mounton;
 allow container_engine_t zero_device_t:chr_file mounton;
+allow container_engine_t container_file_t:sock_file mounton;
+allow container_engine_t container_runtime_tmpfs_t:dir ioctl;
```
Comment (rhatdan) [inline container.te:1459]: allow container_engine_t container_runtime_tmpfs_t:dir list_dir_perms;

Comment (rhatdan) [reply container.te:1459]: AVCs often come in pairs and show up later in testing or use, so it is always better to be a little less tight.

**Comment by haircommander** [comment]
updated! PTAL

**Comment by rhatdan** [comment]
LGTM

**Comment by lsm5** [comment]
problem is at https://pagure.io/fedora-infrastructure/issue/12183

**Comment by lsm5** [comment]
/packit build
/packit test

Ignore f41 and rawhide for now. I guess f39 can be ignored too unless it's critical

**Comment by lsm5** [comment]
> problem is at https://pagure.io/fedora-infrastructure/issue/12183

I removed f41 and rawhide tests from the [branch protection list](https://github.com/containers/container-selinux/settings/branch_protection_rules/34913546) for now.

**Comment by haircommander** [comment]
@rhatdan good to merge?

**Review by lsm5 — APPROVED** [review]
LGTM

**Pull request merged**